.azure-pipelines/azure-pipelines.yml

```diff
@@ -32,55 +32,6 @@ jobs:
       xcodeDeveloperDir: '$(XCODE_PATH)'
       args: 'SYMROOT="$(Build.BinariesDirectory)"'
 
-  - task: Xcode@5
-    displayName: 'Build CrashReporter macOS Framework for Apple Silicon'
```

Comment:
This task was needed because we could build for Apple Silicon using Xcode 12 only, while all others we built with Xcode 11. Separate task not needed as we are not using Xcode 11 anymore.